config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml

```diff
@@ -1,15 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
-    service.binding: path={.metadata.name}-{.spec.users[0].db}-{.spec.users[0].name},objectType=Secret
-    service.binding/connectionString: path={.metadata.name}-{.spec.users[0].db}-{.spec.users[0].name},objectType=Secret,sourceKey=connectionString.standardSrv
-    service.binding/password: path={.metadata.name}-{.spec.users[0].db}-{.spec.users[0].name},objectType=Secret,sourceKey=password
-    service.binding/provider: community
-    service.binding/type: mongodb
-    service.binding/username: path={.metadata.name}-{.spec.users[0].db}-{.spec.users[0].name},objectType=Secret,sourceKey=username
```

Comment:
revert

Reply:
This happens if you have an old version under bin/controller-gen.

My suggestion:
- delete it
- run `make controller-gen` to retrieve the one with the correct version